src/workloads/execution/LastPoint.yml

```diff
@@ -0,0 +1,188 @@
+      OperationName: RunCommand
+      OperationCommand:
+        {create: &coll Collection6, timeseries: {timeField: "time", metaField: "tags"}}
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Nop: true
+  - Nop: true
+  - Repeat: 1
+    Database: *db
+    Collection: *coll
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000
+    BatchSize: &batchSize 100
+    Document:
+      part1: {^Choose: {from: [1, 2, 3, 4]}}
+      part2: {^Choose: {from: [1, 2, 3, 4]}}
+      time: {^RandomDate: {
+        min: {^Inc: {start: 0, multiplier: 1, step: 1}},
+        max: {^Inc: {start: 1, multiplier: 1, step: 1}}
+      }
+      }
+      double: {^Choose: {from: [
+        {^RandomDouble: { min: 0.0, max: 500.0 }},
+        null], weights: [2, 8]}}
+      tags: { value: {^Choose: {from: [
+        {^RandomInt: {min: -100, max: 100}},
+        {^RandomDouble: { min: 0.0, max: 500.0 }},
+        null], weights: [1, 1, 8]}}}
+      string: {^Choose: {from: [
+        {^RandomString: {length: 15}},
+        null], weights: [1, 4]}}
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+
+- Name: CreateIndex
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Nop: true
+  - Repeat: 1
+    Database: *db
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        createIndexes: *coll
+        indexes: [{key: {"tags.value": 1, "time": -1}, name: "x_1"}]
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+
+- Name: Run
+  Type: RunCommand
+  Database: *db
+  Phases:
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Repeat: 10
+    Database: *db
+    Operations:
+    - OperationMetricsName: RunLastPointCommand
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: *coll
+        pipeline:
+          [
+            {$sort: {
+              "tags.value": 1,
+              "time": -1,
+            }},
+            {$group: {
+              _id: "$tags.value",
+              last: {
+                $first: "$$ROOT"
```

Comment:
This is an interesting way of writing the group accumulator - accumulating on the first "$$ROOT" works, but I'm not sure if this would be the way that we would suggest to users to do this. I was thinking more along the lines of what the scope / technical design has, which is having a $first accumulator for each relevant field (ie. `measure1: {$first: {$measure1}}, measure2: {$first: {measure2}} ...`).

Rus, do you know if there's any preference from product on how we would advise users to write lastpoint queries?